README.md

```diff
@@ -1,8 +1,15 @@
+## :warning: Deprecation warning
+In preparation for the KiCad v6 release this repository will be locked down on 2020-09.01. No new pull-requests will be accepted. Existing pull-requests can be worked on until 2020-10-01. Changes breaking v5.1 compatibility can be merged starting 2020-09-01.
```

Comment:
I'd write out 'Sept 1, 2020' and 'Oct 1, 2020`

Reply:
I am a fan of iso-dates, but writing them out like that is better for internationalization. Besides, there was a typo in the first date.